pom.xml

```diff
@@ -130,5 +129,17 @@
             <version>1.0.5</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.jenkins</groupId>
+            <artifactId>configuration-as-code</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
```

Comment:
It should not be needed, because snakeyaml comes from the plugin. Why did you need it?

Reply:
I'd like to move this ahead a little bit, so I have removed this dependency and adjusted the import statement.